### ISSUE TITLE
Styles are not loaded for embedded controls

### ISSUE THREAD

**Issue by jl0pd** [AvaloniaUI/Avalonia#10943]
**Describe the bug**

Avalonia can embed it's controls into WPF view. Unfortunately styles are not loaded starting from preview-5 or 6 (I was unable to run example on 11.0.0-preview5 tag)

**To Reproduce**

1. Go to `samples/interop/WindowsInteropTest`
2. Build
3. Run
4. Select WPF

**Expected behavior**

`AvBtn` and `Avalonia button` groups have a button

**Screenshots**

Preview 4
![preview4](https://user-images.githubusercontent.com/36221916/230566865-32bf5e8e-8add-4244-8090-976cb6a9c698.png)

Preview 6
![preview6](https://user-images.githubusercontent.com/36221916/230566890-6de887c3-409d-4954-ae60-e0129092aa07.png)

**Desktop (please complete the following information):**

- OS: Windows 10 21H2 19044.2728
- Version: preview-6

**Comment by maxkatz6** [comment]
@jl0pd @kekekeks I don't think it's about styles.
Seems like Renderer should be started for each host control individually, which makes sense. I suppose it just worked before with old renderers.
I.e. adding this line fixes rendering of one of the buttons:
```csharp
TopLevel.GetTopLevel((Avalonia.Controls.Button)RightBtn.Content)!.Renderer.Start();
```

**Comment by jl0pd** [comment]
Yeah, it seems to be the case. I've got used to that if control is not rendering then I'm missing StyleInclude somewhere
